src/definitions/modules/dropdown.js

```diff
@@ -2679,7 +2679,11 @@ $.fn.dropdown = function(parameters) {
                 : value,
               newValue
             ;
-            if(hasInput) {
+            if($selected && $selected.hasClass(className.actionable)){
```

Comment:
Here the setting of a value is skipped when it's actionable. This seems logical, but later in `set.selected` the value is always written and only label/text is skipped. (See other review comments)